src/ChargeTrapping/ChargeTrapping.jl

```diff
@@ -6,8 +6,9 @@ function _calculate_signal(
         pathtimestamps::AbstractVector{T}, 
         charge::T,          
         wpot::Interpolations.Extrapolation{T, 3}, 
-        S::CoordinateSystemType
-    ) where {T <: SSDFloat}
+        S::CoordinateSystemType,
+        point_types::Union{PointTypes{T, N, CS}, Nothing} = nothing
+    ) where {T <: SSDFloat, N, CS}
```

Comment:
And then remove it here:
```suggestion
        point_types::PointTypes{T, N, S}
    ) where {T <: SSDFloat, N, S}
```